src/libraries/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -26,8 +26,8 @@
     <Compile Include="System\Diagnostics\SymbolStore\SymLanguageVendor.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="System.IO.FileSystem" />
```

Comment:
When we have a reference from runtime into System.Private.CoreLib all references that are built in the repo need to be P2P (Don't remember exactly the reason) @ericstj could you remember me why?

Reply:
To avoid type-conflicts.  Referencing the ref assembly will include the system.runtime based reference assembly which may have type-conflicts with CoreLib.  If the assembly referenced doesn't have conflicts then its ok.

Reply:
Thanks for reminding me 😄